choosing-cleanser.html

```diff
@@ -40,6 +44,7 @@ <h2>Skinclusive | 28/09/2021</h2>
                 <div class="video">
                     <iframe width="985" height="480" src="https://www.youtube.com/embed/Xwd2heDYy_A" frameborder="0" allowfullscreen></iframe>
                 </div>
+                <p class="credit">Video courtesy of Dr Dray on YouTube</p>

```

Comment:
In my final version, I added another line of text under each video crediting the original creator.